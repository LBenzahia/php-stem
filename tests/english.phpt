--TEST--
Modified English Porter stemming algorithm (extensive)
--SKIPIF--
<?php if (!extension_loaded("stem") || !stem_enabled(STEM_ENGLISH)) print "skip"; ?>
--POST--
--GET--
--FILE--
<?php

require("fulltests.php");

runTest(STEM_ENGLISH, "english.txt");

?>
--EXPECT--
BAD: 0
