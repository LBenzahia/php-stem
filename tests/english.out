'
''
'a
's
aa
as
'
a
s
aa
as
a
a'
a'
aa
aback
abandon
abandon
abandon
abandon
abandon
abas
abash
abat
abat
abbey
abbot